nipype/interfaces/io.py

```diff
@@ -1511,12 +1511,21 @@ class FSSourceOutputSpec(TraitedSpec):
         loc='mri')
     curv = OutputMultiPath(File(exists=True), desc='Maps of surface curvature',
                            loc='surf')
+    avg_curv = OutputMultiPath(
+        File(exists=True), desc='Average atlas curvature, sampled to subject',
+        loc='surf')
     inflated = OutputMultiPath(
         File(exists=True), desc='Inflated surface meshes',
         loc='surf')
     pial = OutputMultiPath(
-        File(exists=True), desc='Gray matter/pia mater surface meshes',
+        File(exists=True), desc='Gray matter/pia matter surface meshes',
```

Comment:
Ah, thanks. Fixed. And hopefully gave reasonable descriptions for the other two.